autogpt_platform/frontend/src/app/profile/page.tsx

```diff
@@ -24,10 +56,76 @@ export default function PrivatePage() {
     return null;
   }
 
+  //TODO: Remove this once we have more providers
+  delete providers["notion"];
+  delete providers["google"];
+
   return (
     <div>
       <p>Hello {user.email}</p>
       <Button onClick={() => supabase.auth.signOut()}>Log out</Button>
+      <div>
+        {/* <Alert className="mb-2 mt-2">
+          <AlertDescription>Heads up!</AlertDescription>
+          <AlertDescription>
+            <p>
+              You need to manually remove credentials from the Notion after
+              deleting them here, see{" "}
+            </p>
+            <a href="https://www.notion.so/help/add-and-manage-connections-with-the-api#manage-connections-in-your-workspace">
+              Notion documentation
+            </a>
+          </AlertDescription>
+        </Alert> */}
```

Comment:
I don't think it's scalable to do this for every single provider that doesn't have a revocation endpoint. A general note saying "Hey, you may also have to delete the connection with AutoGPT in your {provider} settings" would be plenty. Could be here or in the documentation for the platform.

Reply:
I return 501 and displayed information in toast.

Reply:
I realized 501 is a bit weird here because the base request ("delete these credentials") is still succeeding. I changed it to a response object `{ deleted: true; revoked: bool | null }`, where `revoked = null` if the credentials type isn't revocable (e.g. API key).